UnitsNet/UnitDefinitions/AreaMomentOfInertia.json

```diff
@@ -0,0 +1,79 @@
+﻿{
+  "Name": "AreaMomentOfInertia",
+  "BaseUnit": "MeterToTheFourth",
+  "XmlDoc": "A geometric property of an area that reflects how its points are distributed with regard to an axis.",
+  "Units": [
+    {
+      "SingularName": "MeterToTheFourth",
+      "PluralName": "MetersToTheFourth",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "m⁴" ]
```

Comment:
The ⁴ doesn't render in Chrome for Android, I imagine this might be a problem in other software too. Since we've started with the ² in other parts of the code I would suggest to add a ``"m^4"`` in the array of abbreviations as an alternative and keep this as the first one.

(This applies to all instances in this file)